docs/schema.md

```diff
@@ -816,15 +816,21 @@ x-google-marketplace:
 
 Each entry under `resources` is roughly equivalent to a workload in the app.
 
-* `affinity` defines the relationship between the nodes and the replicas.
+*   `requests` defines the desired resource allocations, and is required for each
```

Comment:
Style is only a single space after `*`, right?

Reply:
Updated - I'm not sure what the universal/general best practice is, but leaving as-is (single-space). Internally, autoformatters will correct to having text align at the fourth space.